client/layout/index.jsx

```diff
@@ -368,6 +369,11 @@ export default withCurrentRoute(
 		const isJetpack =
 			( isJetpackSite( state, siteId ) && ! isAtomicSite( state, siteId ) ) ||
 			currentRoute.startsWith( '/checkout/jetpack' );
+		const isReaderStream =
```

Comment:
nit: It's best not to check the currentRoute if possible because we'll want to change the routes when we support locale codes ( but it looks like we probably need to here 🤷‍♂️ )

Reply:
> but it looks like we probably need to here

It seems that way.  🤔  I think we have some helpers for adding/removing locale codes from paths (i haven't used them but pretty sure i saw them at some point), so in that case of locales maybe we update the check to something like ` removeLocaleCode( currentRoute ).startsWith( ... )` ?

Reply:
> when we support locale codes

🤔 is this specifically for logged-out? if so it seems this layout component isn't used in that case. for logged-out layout we could probably do the above.